RoutesApp_ios/utilities/helpers/ConstantVariables.swift

```diff
@@ -18,10 +18,9 @@ class ConstantVariables {
     static let defCityLong = "CityLongitude"
     static let primaryColor = "primary-color"
     static let routeListCell = "RouteListTableViewCell"
-    static let routeTitle = "Route List"
+    static let routeTitle = String.localizeString(localizedString: "Lines")
```

Comment:
as this is a localized string now, isn't it better to just call it when its needed?

Reply:
Corrected :)